tests/cromwell.conf

```diff
@@ -16,7 +16,7 @@ backend {
           --cidfile ${cwd}/execution/docker_cid \
           --user $(id -u $USER) \
           -i \
-          --entrypoint /bin/sh \
+          --entrypoint /bin/bash \
```

Comment:
Is this file even still necessary?

Reply:
Yes. This allows testing on absolute paths. Since this pipeline is used actively on our cluster I want to make sure both absolute and relative paths work